.github/workflows/main.yaml

```diff
@@ -0,0 +1,44 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 15.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install node v${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Restore pnpm's global store
+        uses: actions/cache@v2
+        with:
+          path: ~/runner/.pnpm-store/v3
+          key: ${{ runner.os }}-node-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install pnpm
+        run: npm i -g pnpm
+
+      - name: Install dependencies
+        run: pnpm i
+
+      - name: Lint
+        run: pnpm run lint
+
+      - name: Build dist
+        run: pnpm run build
+
+      - name: Run tests
+        run: pnpm test
```

Comment:
I'm not super familiar with `pnpm`, what is the motivation for migrating to this from `yarn`?

Reply:
Mainly the speed for building. I found in almost all cases pnpm to be up to 2x faster than yarn. https://github.com/pnpm/benchmarks-of-javascript-package-managers